docs/using_eessi/eessi_container.md

```diff
@@ -0,0 +1,165 @@
+```
+source /cvmfs/pilot.eessi-hpc.org/latest/init/bash
+```
+This may take a while as data is downloaded from Stratum 1 server which is part of the
+CernVM-FS infrastructure to distribute files. You should see the following (or
+similar output depending on the CPU architecture of the machine you are accessing
+EESSI).
+```
+Found EESSI pilot repo @ /cvmfs/pilot.eessi-hpc.org/versions/2021.12!
+archspec says x86_64/intel/skylake_avx512
+Using x86_64/intel/skylake_avx512 as software subdirectory.
+Using /cvmfs/pilot.eessi-hpc.org/versions/2021.12/software/linux/x86_64/intel/skylake_avx512/modules/all as the directory to be added to MODULEPATH.
+Found Lmod configuration file at /cvmfs/pilot.eessi-hpc.org/versions/2021.12/software/linux/x86_64/intel/skylake_avx512/.lmod/lmodrc.lua
+Initializing Lmod...
+Prepending /cvmfs/pilot.eessi-hpc.org/versions/2021.12/software/linux/x86_64/intel/skylake_avx512/modules/all to $MODULEPATH...
+Environment set up to use EESSI pilot software stack, have fun!
+[EESSI pilot 2021.12] $ 
+```
+The last line is the prompt. Run `module avail` to see which modules and
+extensions are available. About 280 modules and 1800 extensions are installed.
+Below is a short excerpt from `module avail` showing 10 modules only.
+```
+   PyYAML/5.3-GCCcore-9.3.0
+   Qt5/5.14.1-GCCcore-9.3.0
+   Qt5/5.15.2-GCCcore-10.3.0                               (D)
+   QuantumESPRESSO/6.6-foss-2020a
+   R-bundle-Bioconductor/3.11-foss-2020a-R-4.0.0
+   R/4.0.0-foss-2020a
+   R/4.1.0-foss-2021a                                      (D)
+   re2c/1.3-GCCcore-9.3.0
+   re2c/2.1.1-GCCcore-10.3.0                               (D)
+   RStudio-Server/1.3.1093-foss-2020a-Java-11-R-4.0.0
+```
+Load modules with `module load package/version`, e.g.,
+`module load R/4.1.0-foss-2021a`, and try out the software. See below for a short
+session
+```
+[EESSI pilot 2021.12] $ module load R/4.1.0-foss-2021a
+[EESSI pilot 2021.12] $ which R
+/cvmfs/pilot.eessi-hpc.org/versions/2021.12/software/linux/x86_64/intel/skylake_avx512/software/R/4.1.0-foss-2021a/bin/R
+[EESSI pilot 2021.12] $ R --version
+R version 4.1.0 (2021-05-18) -- "Camp Pontanezen"
+Copyright (C) 2021 The R Foundation for Statistical Computing
+Platform: x86_64-pc-linux-gnu (64-bit)
+
+R is free software and comes with ABSOLUTELY NO WARRANTY.
+You are welcome to redistribute it under the terms of the
+GNU General Public License versions 2 or 3.
+For more information about these matters see
+https://www.gnu.org/licenses/.
+```
+Leave the container with `exit`.
+### Reusing the previous session
+You may have noted the following line in the output of `eessi_container.sh`
+```
+Using /tmp/eessi.RC6kYEjmSj as tmp storage (add '--resume /tmp/eessi.RC6kYEjmSj' to resume where this session ended).
+```
+Just try the following command to "resume" from the last session. _Note, the
+parameter after `--resume` is very likely different for your last session. Just
```

Comment:
`is very likely` -> `*will* be`

Reply:
Changed by https://github.com/EESSI/docs/pull/100/commits/0eae16fe58096e7b995d7e7d2ddc3d9208fc2001